### PR TITLE
Feature: Implement the canvas drawing functionality

### LINKED ISSUE #42: Implement Canvas for drawing objects

### PR DESCRIPTION
**Pull request by logickoder**
Closes #42 

https://github.com/aritra-tech/Notify/assets/42023982/d9714ff1-c609-48f3-8d75-654a2ada754d

### CODE REVIEW

**Comment by logickoder** [comment]
@aritra-tech

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -45,7 +45,7 @@ androidx-biometric = { module = "androidx.biometric:biometric", version.ref = "b
 androidx-core-ktx = { module = "androidx.core:core-ktx", version.ref = "core-ktx" }
 android-compose-bom = { module = "androidx.compose:compose-bom", version.ref = "bom" }
 android-kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlinBom" }
-compose-ui = { module = "androidx.compose.ui:ui" }
+compose-ui = "androidx.compose.ui:ui:1.6.0-alpha08"
```
Comment (aritra-tech) [inline gradle/libs.versions.toml:48]: Can you please write this in proper format?

**Review by aritra-tech — CHANGES_REQUESTED** [review]

File: gradle/libs.versions.toml (hunk 1 above)
Comment (logickoder) [reply gradle/libs.versions.toml:48]: I need to use the Alpha version of the UI lib for the screenshot functionality, as it is not available in stable. So I can't use the bom

Comment (aritra-tech) [reply gradle/libs.versions.toml:48]: Okay got it

**Review by aritra-tech — APPROVED** [review]
LGTM 🚀

**Pull request merged**